src/core/planetsephems/precession.cpp

```diff
@@ -51,7 +54,7 @@ static double c_psi_A=0.0, c_omega_A=0.0, c_chi_A=0.0, /*c_p_A=0.0, */ c_epsilon
 
 static const double arcSec2Rad=M_PI*2.0/(360.0*3600.0);
 
-static const double PQvals[8][5]=
+static const std::vector<std::vector<double>> PQvals=
```

Comment:
Is this really required? `std::vector<std::vector<double>>` is less efficient in terms of memory locality than `double[N][M]`, since every subvector is located in a separate part of the heap. If you really want it to be a C++ container, you should use `std::array<std::array<double,5>,8>`, which will, aside from keeping the data together, also make the sizes compile-time constants.

Similarly for the subsequent instances of such change.